app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/ClassLength
```

Comment:
Yeah, may as well. Not sure how we're going to refactor it...maybe a new controller? *Shrug*